### PR TITLE
chore: update ada handle copies

### PR DESCRIPTION
**Pull request by greatertomi**
# Checklist

- [ ] JIRA - \<link>
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

---
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1888/6077581262/index.html) for [39471e3c](https://github.com/input-output-hk/lace/pull/479/commits/39471e3c47e64829b88b85d38fcb76c02c0fc897)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 28     | 7      | 0       | 0     | 35    | ❌     |

### CODE REVIEW

**Review by VanessaPC — APPROVED** [review]
Nice!

**Review by shawnbusuttil — APPROVED** [review]

**Review by gabriela-ponce — APPROVED** [review]

File: apps/browser-extension-wallet/src/lib/translations/en.json [hunk 1]
```diff
@@ -255,7 +255,7 @@
     "deleteModal.buttons.cancel": "No",
     "reviewModal": {
       "title": "Review change to {{name}}'s handle",
-      "banner.browserDescription": "This handle may no longer belong to your contact {{name}}. Please, review to proceed.",
+      "banner.browserDescription": "This handle may no longer belongs to your contact {{name}}. Please, review to proceed.",
```
Comment (rhyslbw) [inline apps/browser-extension-wallet/src/lib/translations/en.json:258]: This is a grammar error. Can you point me to where this change was requested?

Comment (rhyslbw) [inline apps/browser-extension-wallet/src/lib/translations/en.json:258]:
```suggestion
      "banner.browserDescription": "This handle may no longer belong to your contact {{name}}. Please, review to proceed.",
```

This is a grammar error. @daniloprates can you approve this?

**Review by rhyslbw — CHANGES_REQUESTED** [review]

File: apps/browser-extension-wallet/src/lib/translations/en.json (hunk 1 above)
Comment (daniloprates) [inline apps/browser-extension-wallet/src/lib/translations/en.json:258]: This is wrong. It was fixed on Ditto and will soon be reflected in Figma. Let's please revert back to `belong` please 🙏

**Review by daniloprates — CHANGES_REQUESTED** [review]

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**